src/sss_client/pam_sss.c

```diff
@@ -2472,6 +2472,8 @@ static void eval_argv(pam_handle_t *pamh, int argc, const char **argv,
             }
         } else if (strcmp(*argv, "quiet") == 0) {
             *quiet_mode = true;
+        } else if (strcmp(*argv, "allow_chauthtok_by_root") == 0) {
+            *allow_chauthtok_by_root = true;
```

Comment:
Hi,

thank you for adding the option, please use `flags` instead of a dedicate variable, see the following lines for an example. Additionally, pleas add the new option to the pam_sss man page in ./src/man/pam_sss.8.xml.

bye,
Sumit

Reply:
This option is only used in one place, so I thought a separate variable would be better than flag.

But sure, whatever you feel is better.